### PR TITLE
NSEK-84: Adding pull request workflow by github actions

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "https://localhost:8000"
```
Comment (turegjorup) [inline cypress.json:2]: Is this necessary when we set as `env` value on the e2e job? Also, doesn't this change local/dev behavior?

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+      - name: Yarn install
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - run: yarn install
+      - name: Yarn build
+        run: yarn run encore production
+      - name: Run local server
+        run: symfony server:start --no-tls --port=8000 -d
+      - name: Run Doctrine Migrations
```
Comment (turegjorup) [inline .github/workflows/pr.yaml:119]: Cloud we disable migrations to see if the cypress `sample_spec.js` runs?

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        run: php bin/console doctrine:migrations:migrate --no-interaction
+      - name: Load Fixtures
```
Comment (turegjorup) [inline .github/workflows/pr.yaml:121]: Cloud we disable fixtures to see if the cypress `sample_spec.js` runs?

**Review by turegjorup — CHANGES_REQUESTED** [review]
Added specific comments on code

Some general  notes:
* values and settings specific til running in github action should be set via `env` values in the action file. 
* It should be possible to use `localhost` on Github Actions, but I have experienced cases where `localhost` doesn't resolve. So i recommend `127.0.0.1`  (https://docs.github.com/en/actions/guides/about-service-containers#running-jobs-on-the-runner-machine)

I would like to see the cypress step run. Could we `#` comment the steps with migrations and fixtures until they are needed?

Also - i think you need to ad `"cypress": "^6.5.0"` to `pacakge.json` as a dev dependency. If you don't I think you will get a `Error: Cannot find module 'cypress'` when the cypress part actually runs. (https://github.com/cypress-io/github-action/issues/248)

File: cypress.json (hunk 1 above)
Comment (lats1) [reply cypress.json]: The cypress.json itself is needed cypress will not run if it's missing. I've omitted the baseUrl property, as we use env values instead. See 4a7d28e.

File: .github/workflows/pr.yaml (hunk 3 above)
Comment (lats1) [reply .github/workflows/pr.yaml]: Yup. Done in 1bd1113

**Review by turegjorup — APPROVED** [review]

**Pull request merged**